pkg/eval/platform/is-unix.go

```diff
@@ -0,0 +1,6 @@
+// +build aix darwin dragonfly freebsd illumos linux netbsd openbsd solaris
+
+package platform
+
+var isUnix bool = true
```

Comment:
Leave out the type, and use a `var` block:

```go
var (
    isUnix = true
    isWindows = false
)

Reply:
Done